src/cli-main.ts

```diff
@@ -941,6 +944,42 @@ program
     }
   });
 
+// sync sandbox from remote using jwt
+program
```

Comment:
general comment - should we verify the JWT is valid first?

Reply:
The jwt-decode library that we use below performs validation before decoding.